airbyte-db/lib/src/main/java/io/airbyte/db/instance/jobs/JobsDatabaseSchema.java

```diff
@@ -22,16 +22,23 @@
  */
 public enum JobsDatabaseSchema implements TableSchema {
 
-  ATTEMPTS("Attempts.yaml"),
-  JOBS("Jobs.yaml"),
+  ATTEMPTS("Attempts.yaml", false),
+  JOBS("Jobs.yaml", false),
   AIRBYTE_METADATA("AirbyteMetadata.yaml");
 
   static final Path SCHEMAS_ROOT = JsonSchemas.prepareSchemas("jobs_database", JobsDatabaseSchema.class);
 
   private final String schemaFilename;
+  private final Boolean isPartofImportExport;
 
   JobsDatabaseSchema(final String schemaFilename) {
     this.schemaFilename = schemaFilename;
+    this.isPartofImportExport = true;
+  }
```

Comment:
```suggestion
  JobsDatabaseSchema(final String schemaFilename) {
    this(schemaFilename, true);
  }
```
When overloading a constructor, we prefer to have it call a base constructor instead of having it set member fields itself.